src/main/java/seedu/address/model/loan/Loan.java

```diff
@@ -19,17 +19,31 @@
     private final BorrowerId borrowerId;
     private final LocalDate startDate;
     private final LocalDate dueDate;
-    // returnDate
-    // isReturned method
+    private final LocalDate returnDate;
+    private final int renewCount;
+    private final int remainingFineAmount; // in cents
+    private final int paidFineAmount; // in cents
```

Comment:
Would you consider naming the variables `remainingFineAmountInCents` and `paidFineAmountInCents`so we dont need the comment to clarify? Might be abit long for a name but I feel it might be easier when we are handling this variable next time too!

Reply:
Actually I think I will keep this as is for now since it's an int